app/config.py

```diff
@@ -50,8 +69,8 @@ def get_abs_path(file_path: str):
 
 # Email related settings
 NOT_SEND_EMAIL = "NOT_SEND_EMAIL" in os.environ
-EMAIL_DOMAIN = os.environ["EMAIL_DOMAIN"].lower()
-SUPPORT_EMAIL = os.environ["SUPPORT_EMAIL"]
+EMAIL_DOMAIN = os.environ.get("EMAIL_DOMAIN").lower()
```

Comment:
I think for obligatory variables like `EMAIL_DOMAIN`, it's better to use `[]` to throw an exception (so the app will refuse to start)

Reply:
It's fixed on the latest push

Reply:
Hey I don't see the update here ...

Reply:
but `EMAIL_DOMAIN` is supposed to be a list or just a string?

Reply:
Oh you're right, EMAIL_DOMAIN is just a string indeed.